README.md

```diff
@@ -7,6 +7,17 @@
 Estuary Updater is a micro-service that updates the Neo4j graph database used by Estuary in real-time
 by reading and processing messages from the UMB.
 
+## Dependency Management
+
+To manage dependencies, this project uses [pip-tools](https://github.com/jazzband/pip-tools) so that the production dependencies are pinned and the hashes of the dependencies are verified during installation.
+
+The unpinned dependencies are recorded in **setup.py**, and to generate the **requirements.txt** file, run `pip-compile --generate-hashes --output-file=requirements.txt --allow-unsafe`. This is only necessary when adding a new package. To upgrade a package, use the `-P` argument of the `pip-compile` command.
```

Comment:
Actually, could you abstract this with tox like this:
https://github.com/redhat-exd-rebuilds/estuary-api/pull/303